tests/tests.js

```diff
@@ -116,6 +116,10 @@ describe('react-gettext-parser', () => {
       expect(messages).to.have.length(1);
     });
 
+    it('should not fail on import()', () => {
+      const messages = extractMessagesFromFile('tests/fixtures/DynamicImportCaller.js');
```

Comment:
Could this be more affirmative, like, `supports import()` and then expecting messages to be something?